API.md

```diff
@@ -84,6 +89,18 @@ This will open a browser window that will allow you to authenticate in.
 ##### Response
 The bearer token should be returned and viewable in the browser window after authenticating.
 
+#### iRODS as an OAuth Protected Resource
+While running in this mode, the HTTP API does not provide any grants to which you can authenticate to.
+Instead, you must implement an OAuth client yourself, authenticate, and provide an access token as a
+Bearer token when querying the HTTP API endpoints.
+
+In order for the access token to be accepted, two conditions must be met:
+1) The OpenID Provider recognizes the token as valid and active
+2) The token must be able to be mapped to a user, using either `irods_user_claim` or `user_attribute_mapping`. 
```

Comment:
Missing a period for (1).

The use of parens caught my attention. Did you know it would render as a numbered list? I always thought you had to use "1.".

Reply:
You're right, I just defaulted to org lists instead.

Reply:
github markdown does either, yes.

Reply:
Done. Changed to use '.' version.